econml/sklearn_extensions/linear_model.py

```diff
@@ -1130,6 +1130,8 @@ class WeightedLassoCVWrapper:
     def __init__(self, *args, **kwargs):
         self.args = args
         self.kwargs = kwargs
+        # set model to WeightedLassoCV by default so that cv is accessible pre-fit
+        self.model = WeightedLassoCV(*args, **kwargs)
```

Comment:
This requires some further consideration. There are three models, `WeightedLassoCV`, `WeightedMultiTaskLassoCV` and `WeightedLassoCVWrapper` and it's unclear how the weighted splits interact with the group splits. Right now,  `WeightedLassoCV`, `WeightedMultiTaskLassoCV` modify the `cv` argument into a `WeightedKFold` (or `StratifiedWeightedKFold`) at `fit` time so the `fit_with_groups` method would do nothing since it sets the `cv` argument before fitting.